frontend/app/src/components/StatusWidget/StatusWidget.tsx

```diff
@@ -113,6 +113,11 @@ interface State {
    * shown, even if they'd otherwise be minimized.
    */
   promptHovered: boolean
+
+  /**
+   * True if the running man animation should be displayed after an initial delay.
+   */
+  showRunningMan: boolean
```

Comment:
nit: based on the comment, I think a better variable name would be `delayShowingRunningMan`. Because if the `showRunningMan` is `false`, it is shown right away instead of never, right?

Reply:
Hm, after looking at the rest of the code, it seems like the variable name is correct but the comment is confusing. I think the comment should just be: `If state is true, the running man animation is shown` or similar because the variable itself is controlled by the delay, but not directly related.

Reply:
Thanks, this comment is unclear. I dropped the part about the delay.